modules/user_manual/pages/files/encrypting_files.adoc

```diff
@@ -56,13 +56,14 @@ may enable; see the Recovery Key Password section (below) to learn about this.
 [[sharing-encrypted-files]]
 == Sharing Encrypted Files
 
-Only users who have private encryption keys have access to shared
-encrypted files and folders. Users who have not yet created their
-private encryption keys will not have access to encrypted shared files;
-they will see folders and filenames, but will not be able to open or
-download the files. They will see a yellow warning banner that says
-`Encryption App is enabled but your keys are not initialized, please
-log-out and log-in again.`
+Only users who have private encryption keys have access to shared encrypted files and folders. 
+Users who have not yet created their private encryption keys will not have access to encrypted shared files;
+they will see folders and filenames, but will not be able to open or download the files. 
+They will see a yellow warning banner that says:
+
+""
+Encryption App is enabled but your keys are not initialized, please log-out and log-in again.`
```

Comment:
`have not yet created their private encryption keys` is not congruent with `keys are not initialized`
(created vs. initialized)